modules/metrics-probe/metrics-probe.c

```diff
@@ -146,6 +254,8 @@ metrics_probe_new(GlobalConfig *cfg)
 {
   MetricsProbe *self = g_new0(MetricsProbe, 1);
 
+  _register_global_initializers();
```

Comment:
[opt]
We could place this into the module's init function so that it would run less frequently.
Or alternatively, we could place it into `_init()`, which avoids the allocation if the parser is declared but never used.

Reply:
Good idea, moved to `_init()`.